compiler/record-hessian/src/HessianComputer.cpp

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
plz add unit test files for each *.cpp as *.test.cpp
with count(positive) < count(negative),
where negative unit test name ends with `_NEG`

Reply:
I will add test codes after the overall flow of the code has been determined.  https://github.com/Samsung/ONE/pull/13487#discussion_r1703455165